.github/workflows/checks.yml

```diff
@@ -32,6 +34,18 @@ jobs:
           java-version: 11
       - working-directory: ./scalafix
         run: sbt test
+  dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: cache SBT
+        uses: coursier/cache-action@v6
+      - name: Java 11 setup
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+      - run: sbt "; undeclaredCompileDependenciesTest; unusedCompileDependenciesTest"
```

Comment:
@RustedBones what plugin is this?

Reply:
We already declared the `sbt-explicit-dependencies` [plugin](https://github.com/spotify/scio/blob/3776382a70a61a8c8fa5fa856a67e26ae8ccb387/project/plugins.sbt#L6) but we were not using it in CI